azure-kusto-data/azure/kusto/data/request.py

```diff
@@ -75,15 +100,31 @@ def __init__(self, connection_string):
 
         for kvp_string in connection_string.split(";"):
             key, _, value = kvp_string.partition("=")
-            self[key] = value
+            keyword = self.ValidKeywords.parse(key)
+            if keyword.is_str_type():
+                self[keyword] = value.strip()
+            if keyword.is_bool_type():
+                if value.strip() in ["True", "true"]:

```

Comment:
how about we split it out to a collection (say, `truthy_values`) places in `helpers`

Reply:
also, should we split this to `def ensure_type(key,value):`?

Reply:
Let's leave it here for now, and once we have more usage we can rethink where to locate it.